org.msweet.htmldoc.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.msweet.htmldoc
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: htmldoc
+rename-icon: htmldoc
+rename-desktop-file: htmldoc.desktop
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  #- --socket=wayland
```

Comment:
```suggestion
```

Reply:
Since FLTK 1.4 will support Wayland, I want to leave this in, commented out.

Reply:
but you don't want to "clutter" your repository with standard appstream file. ok.

Reply:
Again, until trying to package HTMLDOC with Flatpak (as a service to users that are only able to install Flatpak applications on shared servers) I had never heard of appstream files. They aren't required for Linux software in general or for any other distribution mechanism, and until that isn't the case I am not adding the file to HTMLDOC itself.

That said, this Wayland stuff is part of the (limited) Flatpak documentation, so I included it thinking it might be necessary. When FLTK 1.4 is released later this year I will need to uncomment that line for Linux systems that support Wayland, and I don't want to have to find that example again. I'll already have enough of a hassle to figure out how to support both X11 and Wayland-based systems... :/